apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java

```diff
@@ -234,6 +234,24 @@ public static int getActiveCount() {
 
     public static DbIterator<Shuffling> getActiveShufflings(int from, int to) {
         return shufflingTable.getManyBy(new DbClause.NotNullClause("blocks_remaining"), from, to, " ORDER BY blocks_remaining, height DESC ");
+        /*return shufflingTable.getManyBy(
```

Comment:
Clean up commented code if it's not needed.

Reply:
done